textdb/textdb-textql/src/main/java/edu/uci/ics/textdb/textql/statements/Statement.java

```diff
@@ -54,6 +59,40 @@ public void setId(String id) {
     }
     
     
+    /**
+     * Return the ID to which a { @code LinkBean } should be connected in order to connect to the input 
+     * of this operator when converted to beans.
+     * @return The ID to be used as input of this operator by a { @code LinkBean }.
+     */
+    public abstract String getInputID();
+    
+    /**
+     * Return the ID to which a { @code LinkBean } should be connected in order to connect to the output
```

Comment:
Change the comments to:

Return the ID of the "last" node in the statement that is connected to the next statement.